controllers/nstemplateset/tier_template_revision_test.go

```diff
@@ -0,0 +1,79 @@
+package nstemplateset
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	testmem "github.com/codeready-toolchain/member-operator/test"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func createTierTemplateRevision(templateRef string) *toolchainv1alpha1.TierTemplateRevision {
+	return &toolchainv1alpha1.TierTemplateRevision{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      templateRef,
+			Namespace: test.HostOperatorNs,
+		},
+	}
+}
+
+func TestGetTTR(t *testing.T) {
+	// given
+	ttRev := createTierTemplateRevision("test-ttr")
+	ttRevTest := createTierTemplateRevision("test-ttr-abc")
+	ttRevExtra := createTierTemplateRevision("test-ttr-def")
```

Comment:
minor - those do not seem to be used , unless I'm missing something.

Reply:
I just kept them to check that out of all the TTR present it matches and gets the required TTR only

Reply:
I see, thanks for explaining.